react/ui-components/src/hoc/app-config-screen-builder/AppLocalisationWrapper.js

```
@@ -0,0 +1,153 @@
+      return state;
+    case "ADD_MISSING_KEY":
+      if (state.find((item) => item.code === action.payload.code)) {
+        return state;
+      } else {
+        const newEntry = action.payload.enabledModules.reduce((acc, locale) => ({ ...acc, [locale.value]: "" }), {
+          code: action.payload.code,
+        });
+        return [...state, newEntry];
+      }
+    case "UPDATE_LOCALIZATION":
+      const checkCodeAlreadyPresent = Array.isArray(state) && state?.some((item) => item?.code === action?.payload?.code);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)